src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1329,6 +1329,10 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
+    
+    <!-- BinderOpt and UseSetterForDefaultArgumentGeneration are added as a temporary solution for IOperation implementation and should probably be removed in the future -->
```

Comment:
This is similar to the current approach used for BoundIndexerAccess

See https://github.com/mavasani/roslyn/blob/07421be2f2618cae7226438c06401fe0a40d7553/src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml#L1505. @genlu for any context.